### DISCUSSION TITLE
New Wrapper looks Interesting... (printFlow)

### DISCUSSION THREAD

**Discussion by inspectionsbybob** [supermerill/SuperSlicer#3983] [General]
There was a new Slicer released recently, or so I thought...
PrintFlow proposes to be the powerful but simple slicer with all the power of the Big slicers with less confusing menus... but for a Fee.  I am fine with that if it works.
They only support a few printers now... but one was close enough to what I have that I thought I would try it.

Lots of Calibration needed... But much simpler than other calibration routines.  About 6 hours worth of Models needed.

I opened up one of the Gcode files, and it said, "Generated by SuperSlicer...."

FANTASTIC...  maybe some of the money will trickle down to the SS team as well, or -ARE THEY- the superslicer team?

It has issues as it is a bit -TOO- simple currently... and that they insist on validating every filament and printer before it will build a profile makes it SLOW to adopt...

Anyway,... anyone else have comments on PRINTFLOW 
https://app.getprintflow.com

**Comment by darco415** [comment]
hmm, sound like SS is going commercial...

**Comment by inspectionsbybob** [comment]
If they make it work they way they say, I would pay!

I managed to get one printer setup, my ender3 clone, and it produced some
really nice prints.

More importantly, I was able to open up the gCode file and see what the
settings were... W O W were they different from what I expected.

The big one was the line width, it was  125% of my nozzle... on the infill
was less than my nozzle (75% or so).  Speeds and acceleration (accel &
jerk) were also much higher than I expected.

What I am NOT happy with is their slowness to add printers and filaments...

And the forum hasn't been touched in several months... not good signs...

I want to support them, but they have to show me they are more than a
flash...

Oh.... and the Gcode says SuperSlicer 2.4... several revisions back...

-----------------------------------------------------------------------------------
Need a quick answer? Call our booking line at (301) 208-8289, or book your
inspection on our website 24/7.
Don't wait for a response as someone else may get the spot you want... call
Today!
-----------------------------------------------------------------------------------
Bob Sisson, ACI, BVI
Inspections by Bob, LLC
www.inspectionsbybob.com
***@***.***
MD Lic#29666; ASHI Member #212016

**Reply by supermerill** [reply]
mmm.
I don't see where the source code of the software is.
If they use an agplv3 library, they have to open their source...

If the webapp can use multiple slicer software and use them with the command line, i guess it's maybe possible to not open-source the web app.

**Reply by supermerill** [reply]
I asked them, and yes, they're using the cli.
I'll see if it's possible to collaborate.

**Comment by inspectionsbybob** [comment]
Since they are offering a SERVICE maybe they don't think....


Bob Sisson

**Comment by inspectionsbybob** [comment]
I have been asking about adding printers and filaments and have gotten -NO RESPONSE-, not even an acknowledgment, so I will likely -NOT- be subscribing until they get their act together.  

Their calibration routines are the BEST I have seen. Better than Prusa, Cura or Teaching Tech.  The retraction test is the only one that still needs a touch of something.

What I really want is a HYBRID where 95% is Printflow, but I have the OPTION of downloading the Project file to make final tweaks if needed.  (Paint on supports/blockers come to mind)
